src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -705,7 +719,10 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
 
             //PROTOTYPE: should be a wellknown member?
             var ctor = type.UnderlyingTupleType?.InstanceConstructors.FirstOrDefault();
```

Comment:
I agree, but I will leave this one for a later change. PROTOTYPE marker is there.